### ISSUE TITLE
Securing "$" token during BibTeX export.

### ISSUE THREAD

**Issue by Zugrik** [mkucej/i-librarian#146]
Hello Martin!

There is a small issue with "$" token. If, say, article title has some TeX commands, Librarian secures closing $ tokens to \$. After exporting library to bibtex file, TeX couldn't recognize them and force me to manually change \$ back to just $. Is this securing a php demand, or we can just live without it?

Thanks,
Eugene.

**Comment by mkucej** [comment]
Please see this comment and discussion why $ is escaped.

https://github.com/mkucej/i-librarian/issues/47#issuecomment-199388230

If you disagree, we can revert that.

**Comment by mkucej** [comment]
Is there a way to distinguish a $ character from a $ in a Latex command?

**Comment by Zugrik** [comment]
I don't think so. $ is a special character in Latex, so if you don't secure it in Latex command with "\\", it will be treated as an exit from math mode. Also, please read my following comment.

**Comment by Zugrik** [comment]
> Please see this comment and discussion why $ is escaped.
> 
> [#47 (comment)](https://github.com/mkucej/i-librarian/issues/47#issuecomment-199388230)
> 
> If you disagree, we can revert that.

Also looked through this discussion. Actually, guys didn't show any arguments, why $ character should be secured. On the other hand, I see that Latex can't cope with \$ correctly. I think the issue is in following: Latex treat "\\$" as it should - placing the very "$" character. But, e.g. in title "$" opens up a math mode, where "_", "^" and other special commands will work by placing sub-, superscripts and so on. Since Latex already treated "\\$" as ordinary "$" character and added it in text, it wouldn't enter the math mode and following "math" input will produce errors.

Hope it helped)

Best,
Eugene.
